.github/workflows/rust.yml

```diff
@@ -16,7 +16,7 @@ jobs:
   run_checks:
     strategy:
       matrix:
-        rust_toolchain_version: ["1.71", "1.72", "1.73", "1.74"]
+        rust_toolchain_version: ["1.71", "1.72", "1.73", "1.74", "1.75"]
```

Comment:
are we using some sort of very-unstable features? IMO this list should be `[MSRV, "stable", "beta"]` if possible. (it's nice for upstream to catch things quickly if community projects use betas)

Reply:
I did not know we could use this list. Good to know!

> are we using some sort of very-unstable features?
No